src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx

```diff
@@ -41,7 +43,12 @@ export const JobDetailAbout: React.FC<JobDetailAboutProps> = ({ isUser = true, s
   const [openApply, setOpenApply] = useState(false);
   const [openExternalApply, setOpenExternalApply] = useState(false);
   const [openAlert, setOpenAlert] = useState(false);
-  const url = window.location.href;
+  let url = window.location.href;
+  const extractedString = url.slice(url.indexOf('jobs'));
+  url =
+    isTouchDevice() && config.env === 'development'
+      ? `https://webapp2.dev.socious.io/${extractedString}`
```

Comment:
no this code is not acceptable solution we don't use urls beyond our config , what is the problem to resolve it like this ?

Reply:
@jeyem this `window.location.href` would return `capacitor://capacitor.native` url. which wouldn't make more sense.